solutions/instances/variables.tf

```diff
@@ -229,6 +229,20 @@ variable "enable_metrics_routing_to_cloud_monitoring" {
   default     = true
 }
 
+variable "metrics_router_settings" {
+  type = object({
+    permitted_target_regions  = optional(list(string))
+    primary_metadata_region   = optional(string)
+    backup_metadata_region    = optional(string)
+    private_api_endpoint_only = optional(bool, false)
+    default_targets = optional(list(object({
+      id = string
+    })))
+  })
+  description = "Global settings for Metrics Routing"
```

Comment:
This will need a markdown helper doc

Reply:
I'll proceed with merging this PR and will create an issue to add the markdown in a subsequent PR

Reply:
https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/issues/301